### PR TITLE
Catchpoint Integration

### PR DESCRIPTION
**Pull request by rcastley**
Catchpoint integration for DataDog (Version 1)

### CODE REVIEW

**Comment by irabinovitch** [comment]
@rcastley Thanks for putting this together.  Would you be able to update metrics.csv to list the metrics your integration would push?

File: Catchpoint/description.html [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+      </div>
+    </div>
+
+    <div class="integration_content" id="int-overview">
+      <h2>Overview</h2>
+      <p>Connect to Catchpoint to:</p>
+      <ul>
+        <li>Configure comprehensive alerts in your event stream</li>
+        <li>Direct links to Analysis Charts in the Catchpoint Portal</li>
+        <li>Alert Type tags for easy filtering</li>
```
Comment (irabinovitch) [inline Catchpoint/description.html:32]: Will the integration include any metrics, or just events?

Comment (rcastley) [inline Catchpoint/description.html:32]: HI Ilan,

The integration only includes events at this point in time.

Cheers,
- Robert

Comment (irabinovitch) [inline Catchpoint/description.html:32]: Thanks. In that case, lets remove all the postgres metrics from metrics.csv.

Comment (rcastley) [inline Catchpoint/description.html:32]: Done!

**Pull request closed without merging**